test-framework/src/runner.rs

```diff
@@ -236,3 +237,13 @@ pub fn run_tests<T: Runner>(runner: &T, expectation_category: &str) {
 
     std::env::remove_var("LEO_TESTFRAMEWORK");
 }
+
+/// Cleans the outputs by removing local paths.
+/// Needed for failing import tests as they require correct setting of CWD
+/// which leads to full system paths being printed out to the test expectations.
+pub fn sanitize_output(out: &String) -> String {
```

Comment:
```suggestion
pub fn sanitize_output(out: &str) -> String {
```
Usually, `&str` is preferred as you don't actually benefit from `.capacity()` and such.